game/src/sandbox/gameplay/census.rs

```diff
@@ -38,15 +55,19 @@ impl State<App> for CensusGenerator {
                 }
                 "Generate" => {
                     // Generate this from self.panel
-                    let config = Config::default();
+                    let mut config = Config::default();
+                    let walk_shorter_than = self.panel.spinner("walk shorter than") as usize;
+                    let walk_bike_shorter_than = self.panel.spinner("walk bike shorter than") as usize;
+
+                    config.walk_for_distances_shorter_than = Distance::miles((walk_shorter_than as f64 / 10 as f64) as f64);
```

Comment:
Lots of casting! `panel.spinner()` hands back an `isize` (signed integer), so I think you could immediately cast to `f64` a few lines up, then use `10.0` to tell Rust the constant is floating pt, and then you don't need the final cast. So this line should just wind up being `Distance::miles(walk_shorter_than / 10.0)`